app/models/user.rb

```diff
@@ -2,6 +2,7 @@ class User < ApplicationRecord
   has_many :comments
   has_many :likes
   has_many :posts
```

Comment:
- [OPTIONAL] As it is a [requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md) for this project, " `For the User model:
Name must not be blank.` " please make sure the user name is not left blanc, please kindly fix that